share/locale/en.json

```diff
@@ -26,10 +26,10 @@
     "systray_download_completed_message": "The user finished downloading your files",
     "systray_download_canceled_title": "OnionShare Download Canceled",
     "systray_download_canceled_message": "The user canceled the download",
-    "help_local_only": "Do not attempt to use tor: for development only",
+    "help_local_only": "Do not attempt to use Tor: For development only",
     "help_stay_open": "Keep onion service running after download has finished",
     "help_shutdown_timeout": "Shut down the onion service after N seconds",
-    "help_transparent_torification": "My system is transparently torified",
+    "help_transparent_torification": "My system is transparently Torified",
```

Comment:
Actually, the whole string `help_transparent_torification` is a relic from the past and isn't in use in the app anymore. This string can be removed completely. I just opened an issue to remove all such strings #681, but in the meantime you can just delete this one in this PR.